tgbot.py

```diff
@@ -177,28 +200,33 @@ def filter_reply(bot , update):
 			message.reply_text(rows[0][1])
 	connection.close()
 
-def main():
-	# Added all the essential command handlers 
-	
-	updater = Updater(TOKEN)
-	dp = updater.dispatcher
-	dp.add_handler(CommandHandler('start' ,startmessage))
-	dp.add_handler(CommandHandler('sendlogo' ,sendlogo))
-	dp.add_handler(CommandHandler('addmeeting' ,add_meeting))
-	dp.add_handler(CommandHandler('next' ,get_list))
-	dp.add_handler(CommandHandler('filter',filter_resp))
-	dp.add_handler(CommandHandler('reply',filter_reply))
-	dp.add_handler(CommandHandler('thonkify',thonkify))
-	dp.add_handler(CommandHandler('help',helpbot))
-	dp.add_handler(CommandHandler('social',add_socialmedia))
-	dp.add_handler(CommandHandler('remove',remove_user))
-	dp.add_handler(CommandHandler('facebook', getlink))
-	dp.add_handler(CommandHandler('github', getlink))
-	dp.add_handler(CommandHandler('website', getlink))
-	dp.add_handler(CommandHandler('meetup', getlink))
-	dp.add_handler(MessageHandler(Filters.status_update, welcome_message))
-	updater.start_polling()
-	updater.idle()
-
+def main(TOKEN):
+        try:
+                updater = Updater(TOKEN)
+                dp = updater.dispatcher
+                COMMAND_HANDLER={'start':startmessage,
+                                 'sendlogo':sendlogo,
+                                 'addmeeting':add_meeting,
+                                 'next':get_list,
+                                 'filter':filter_resp,
+                                 'reply':filter_reply,
+                                 'thonkify':thonkify,
+                                 'help':helpbot,
+                                 'social':add_socialmedia,
+                                 'remove':remove_user,
+                                 'facebook':getlink,
+                                 'github':getlink,
+                                 'website':getlink,
+                                 'meetup':getlink}
+                for key,value in COMMAND_HANDLER.items():
+                        dp.add_handler(CommandHandler(key ,value))
+                dp.add_handler(MessageHandler(Filters.status_update, welcome_message))
+                updater.start_polling()
+                updater.idle()
+        except:
+                print("Token Is Invalid Please Use Correct Token key in config.ini")
```

Comment:
change to: `print('The Token Is Invalid! Use Correct Token in config.ini file')`